src/sentry/models/dashboard.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.constants import ObjectStatus
+from sentry.db.models import BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr
+
+
+class Dashboard(Model):
+    """
+    A dashboard.
+    """
+    __core__ = True
+
+    title = models.CharField(max_length=255)
+    owner = FlexibleForeignKey('sentry.User')
```

Comment:
Are dashboards tied to specific users, or meant to just be organization wide?

Reply:
That's a really good question. I don't really know. @billyvg might have more context

Reply:
Yeah that's one of my questions too, @eurica any ideas?

Reply:
Long term the standard model is that I can publish or share my stuff.  But short term, what do we do with saved searches, etc?

Reply:
Saved searches belong to the project they're created on currently, and everyone with access to that project sees them.

Reply:
Currently discover saved queries are saved to org, but I think it might be helpful to have a created by if we wanted to say, sort list with your saved queries

Reply:
So maybe changing this to created by instead